### PR TITLE
Changes objectives code slightly/better objectives (dont perma me)

### PR DESCRIPTION
**Pull request by HaultyAnonie**
:cl:
add: RCD to steal objective
code: made code to escape the shuttle to then a train 
/:cl:

### CODE REVIEW

**Comment by Tsurupeta** [comment]
Your code has changed my decision and I'm now in full support of your community ban.

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -446,10 +447,10 @@ var/global/list/all_objectives = list()
 		explanation_text = "Free Objective"
 	return target
 
-/datum/objective/rev/find_target_by_role(role, role_type=0)
-	..(role, role_type)
+datum/objective/rev/find_target_by_role(role, role_type=0)
```
Comment (aidan-bowman) [inline code/game/gamemodes/objective.dm]:
```suggestion
/datum/objective/rev/find_target_by_role(role, role_type=0)
```
lol, lmao even

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -446,10 +447,10 @@ var/global/list/all_objectives = list()
...
+ 	..(role, role_type)
```
Comment (aidan-bowman) [inline code/game/gamemodes/objective.dm]:
```suggestion
	..(role, role_type)
```
why did you fucking add a space

File: code/game/gamemodes/objective.dm [hunk 3]
```diff
@@ -446,10 +447,10 @@ var/global/list/all_objectives = list()
...
 	if(target && target.current)
-		explanation_text = "Assassinate, capture or convert [target.current.real_name], the [!role_type ? target.assigned_role : target.special_role]."
-	else
+ 		explanation_text = "Assassinate, capture or convert [target.current.real_name], the [!role_type ? target.assigned_role : target.special_role]."
```
Comment (aidan-bowman) [inline code/game/gamemodes/objective.dm]:
```suggestion
		explanation_text = "Assassinate, capture or convert [target.current.real_name], the [!role_type ? target.assigned_role : target.special_role]."
```

File: code/game/gamemodes/objective.dm [hunk 4]
```diff
@@ -446,10 +447,10 @@ var/global/list/all_objectives = list()
...
+ 	else
```
Comment (aidan-bowman) [inline code/game/gamemodes/objective.dm]:
```suggestion
	else
```

**Review by aidan-bowman — COMMENTED** [review]
how do you fuck up this badly on such a small pr lmao

File: code/game/gamemodes/objective.dm (hunk 4 above)
Comment (HaultyAnonie) [reply code/game/gamemodes/objective.dm]: DO I LOOK LIKE I KNOW

**Comment by HaultyAnonie** [comment]
> Your code has changed my decision and I'm now in full support of your community ban.

HEY!!!

**Comment by TheDarkElites** [comment]
This is unironically the funniest shit I have ever seen

**Comment by checkraisefold** [comment]
what the fuck is this pr

**Review by ivanmixo — CHANGES_REQUESTED** [review]
proper cl please

**Comment by HaultyAnonie** [comment]
> proper cl please

fixed... (i hope)

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**